lib/MusicBrainz/Server/Controller/Admin/Attributes.pm

```diff
@@ -93,6 +93,8 @@ sub edit : Chained('attribute_base') Args(1) RequireAuth(account_admin) {
     my $form = $c->form( form => $form_name, init_object => $attr );
 
     if ($c->form_posted && $form->process( params => $c->req->params )) {
+        $form->field("entity_type")->value( $attr->entity_type );
```

Comment:
There's a small problem: disabled fields aren't submitted by the browser, so if an error occurs in another field (try entering non-numeric characters in child order), the page will reload and the `<select>` will have no selection. To fix that, I think this should go above `$form->process`.